tests/python/frontend/onnx/test_forward.py

```diff
@@ -4278,6 +4278,33 @@ def test_wrong_input():
         relay.frontend.from_onnx(model, shape=wrong_shape_dict)
 
 
+def test_aten():
+    torch.set_grad_enabled(False)
+
+    def _convert_to_onnx(model, inputs):
+        file_name = "{}.onnx".format("aten_model")
+        torch.onnx.export(
+            model, inputs, file_name, export_params=True, verbose=False, opset_version=10
+        )
```

Comment:
Can we assert that this actually contains the expected aten model? What conditions mean that pytorch instead exports a valid onnx graph?

Reply:
https://github.com/pytorch/pytorch/blob/release/1.7/torch/onnx/__init__.py#L72-L74

Reply:
I added `aten=True` which makes all nodes in the graph `aten` rather than just embedding_bag so I've also added a few extra converters to the dispatch.

Reply:
Actually it looks like the version of torch we're using for CI doesnt yet support `aten`. For now we'll have to add this without `aten=True` but we can enable it later if we update versions. The good news is old versions of torch always use ATen fallback ops so we'll be testing embeddingbag.

Reply:
> Actually it looks like the version of torch we're using for CI doesnt yet support `aten`. For now we'll have to add this without `aten=True` but we can enable it later if we update versions. The good news is old versions of torch always use ATen fallback ops so we'll be testing embeddingbag.

Really? The link I posted above is for PT v1.7 and this is what we run on CI.

Reply:
Yeah its kind of goofy, `aten` is present but always causes an error with `1.7.0`. You're right that its suggested replacement works fine. I've added that in the latest commit.